### PR TITLE
Aqua squad for lvmo test cases

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

**Review by shyRozen — DISMISSED** [review]

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -1544,6 +1544,7 @@
 # the pattern "/registry/" match the test path and so the test belongs to
 # Magenta squad.
 SQUADS = {
+    "Aqua": ["/lvmo/"],
```
Comment (petr-balogh) [inline ocs_ci/ocs/constants.py:1547]: https://github.com/red-hat-storage/ocs-ci/blob/master/pytest.ini#L45
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/marks.py#L401

We can add here as well for marker usage

**Comment by petr-balogh** [comment]
What about code owners @vavuthu

**Comment by vavuthu** [comment]
> What about code owners @vavuthu

done but I see errors in codeowners file.

**Review by petr-balogh — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Comment by petr-balogh** [comment]
> > What about code owners @vavuthu
> 
> done but I see errors in codeowners file.

Had to add the team to this ocs-qe:
https://github.com/orgs/red-hat-storage/teams/ocs-qe/teams

**Pull request merged**